core-processor/src/processing.rs

```diff
@@ -356,6 +359,16 @@ pub fn process_success(
         });
     }
 
+    // Must be handled after processing programs creation.
```

Comment:
It needs to be considered what to do if created program with inexistent code id (gr_create_program + gr_pay_rent)

Reply:
Maybe just refund on notes handling would work (provide test for it please)

Reply:
discussed